src/main/java/tornadofx/Lib.kt

```diff
@@ -221,7 +221,7 @@ fun Clipboard.putString(value: String) = setContent { putString(value) }
 fun Clipboard.putFiles(files: MutableList<File>) = setContent { putFiles(files) }
 fun Clipboard.put(dataFormat: DataFormat, value: Any) = setContent { put(dataFormat, value) }
 
-inline fun <T> ChangeListener(crossinline listener: (observable: ObservableValue<out T>?, oldValue: T, newValue: T) -> Unit): ChangeListener<T> =
+inline fun <T> ChangeListener(crossinline listener: (observable: ObservableValue<out T>?, oldValue: T?, newValue: T?) -> Unit): ChangeListener<T> =
```

Comment:
`<T>` has default upper bound and Kotlin generics default upper bound is nullable `Any?`, isn't it?

Reply:
But the type comes from Java which doesn't check null, so either value from Java can be null in various listeners, it seems safer to make this explicit rather than implicit